docs/Usage.md

```diff
@@ -0,0 +1,323 @@
+# Scality COSI Driver Usage Guide: Bucket Provisioning & Access Control
+
+This document provides an overview and step-by-step guidance for implementing **Bucket Provisioning** (both Greenfield and Brownfield) and **Access Control** using the Scality COSI Driver. Example YAML manifests can be found in the [cosi-examples](../cosi-examples/) folder.
+
+> **Note**
+> The Scality COSI Driver supports standard AWS S3 and IAM compliant storage solutions like Scality RING, Scality ARTESCA and AWS S3 & IAM
```

Comment:
```suggestion
> *[!NOTE]
>
> The Scality COSI Driver supports standard AWS S3 and IAM compliant storage solutions like Scality RING, Scality ARTESCA and AWS S3 & IAM
```

> The initial syntax using e.g. \*\*Note\*\* isn't supported any longer.
 
https://github.com/orgs/community/discussions/16925

Reply:
> [!NOTE]
> note highlight

Reply:
Changed for all notes. thanks :)